daemons/mrpd/mrpd.c

```diff
@@ -331,6 +331,7 @@ int process_ctl_msg(char *buf, int buflen, struct sockaddr_in *client)
 		return mvrp_recv_cmd(buf, buflen, client);
 		break;
 	case 'S':
+	case 'I':
```

Comment:
Would you mind adding the following lines from `daemons/mrpd/msrp.c` to the comment above this code?

```
* I+S   Add a stream id to the talker stream id list
* I-S   Remove a stream id from the talker stream id list
* I-A   Remove all stream ids from the interesting talker and listener stream id lists
```

I think those comment blocks are currently the primary documentation for the protocol, so it would be good to keep them up-to-date.

Otherwise, looks good!